react/Viewer/styles.styl

```diff
@@ -17,7 +17,7 @@ $viewerMarginTopMedium = $toolbarHeightMedium - $footerHeight
     background      var(--charcoalGrey)
     color           var(--white)
 
-    +small-screen()
+    +medium-screen()
         color var(--charcoalGrey)
```

Comment:
Not part of this PR, but this color and the background should not defined as absolute colors.

Reply:
Nous sommes pour l'instant en couleur absolue sur le Viewer